PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/Laundry.java

```diff
@@ -0,0 +1,52 @@
+package com.pennapps.labs.pennmobile.classes;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by Jason on 10/21/2015.
+ */
+public class Laundry implements Parcelable{
```

Comment:
perhaps call this LaundryRoom to make it more clear? it seems like a LaundryHall (like Harrison) has many LaundryRooms (on each floor), each of which contain LaundryMachines.